tests/condensation/__init__.py

```diff
@@ -0,0 +1,6 @@
+from action import *
+from cloud import *
+from flavor import *
+from group import *
+from node import *
+from vm import *
```

Comment:
I would recommend importing modules only when they are needed. `import *` may result in unexpected behavior in case there is more than one module with the same name in namespace. So my suggestion here – remove all `from <module> import *`, and only import modules when and where they are really need to be imported.